### ISSUE TITLE
Lower result in PTB

### ISSUE THREAD

**Issue by HITalexwang** [tensorflow/models#420]
## Please let us know which model this issue is about (specify the top-level directory)
I tried to use the released model (Parsey McParseface) to parse PTB test set but only got 92.74% from the eval metric output of this parser. I also checked the output file with the conll evaluation script and got 92.62% for UAS and 85.43% for LAS without punc. This results are lower than those reported in the github of syntaxnet. That's why I'm seeking help from you.
I used the script at syntaxnet/demo.sh and changed it to read and write to files. Here is my demo.sh:

        PARSER_EVAL=bazel-bin/syntaxnet/parser_eval 
        MODEL_DIR=syntaxnet/models/parsey_mcparseface 
        [[ "$1" == "--conll" ]] && INPUT_FORMAT=stdin-conll || INPUT_FORMAT=ptb-test-conll 
        $PARSER_EVAL \ 
         --input=$INPUT_FORMAT \ 
         --output=ptb-test-tag \ 
         --hidden_layer_sizes=64 \ 
         --arg_prefix=brain_tagger \ 
         --graph_builder=structured \ 
         --task_context=$MODEL_DIR/context.pbtxt \ 
         --model_path=$MODEL_DIR/tagger-params \ 
         --slim_model \ 
         --batch_size=1024 \ 
         --alsologtostderr \ 
         | \ 
         $PARSER_EVAL \ 
         --input=ptb-test-tag \ 
         --output=ptb-test-out \ 
         --hidden_layer_sizes=512,512 \ 
         --arg_prefix=brain_parser \ 
         --graph_builder=structured \ 
         --task_context=$MODEL_DIR/context.pbtxt \ 
         --model_path=$MODEL_DIR/parser-params \ 
         --slim_model \ 
         --batch_size=1024 \ 
         --alsologtostderr \
Where the ptb-test-conll is defined in syntaxnet/models/parsey_mcparseface/context.pbtxt ,and is the path to the test set of PTB.
I ran the script by:

        syntaxnet/demo.sh --input=ptb-test-conll --output=ptb-test-out

 I used version 3.3.0 of the converter to convert the data. By the way, the test data include 2416 sentences and 56684 tokens. 
The test data is here
https://github.com/HITalexwang/PTB-dependency

Do you have any idea about this problem?
Or can you parse the test data with the released model and tell me if my result is right?

**Comment by calberti** [comment]
Hi HITalexwang,

You're using a version of the PTB with an older tokenization.  The data you sent includes tokens like "500-stock", where new versions of the PTB split that in three tokens "500", "-", "stock", similarly to OntoNotes.

If you really want to use the old tokenization we can send you a model trained only on the PTB with the old tokenization.  That would correspond to the "WSJ" results in https://arxiv.org/pdf/1603.06042v2.pdf

**Comment by HITalexwang** [comment]
It would be great if you can send me the model trained with older-tokenized PTB. 
Please send it to my email
hitalexwang@gmail.com
@calberti

**Comment by HITalexwang** [comment]
We are using the the LDC1999T42 PTB-3, which is the newest version as far as I am concerned. Please tell me what exactly version of PTB did you use to train the model !
@calberti

**Comment by slavpetrov** [comment]
There are newer releases of the Wall Street Journal portion of the Penn Treebank as part of OntoNotes (https://catalog.ldc.upenn.edu/LDC2013T19) and then as also a standalone corpus (https://catalog.ldc.upenn.edu/LDC2015T13). The appendix in our ACL'15 paper provides the details regarding the Treebank Union setup that underlies the released Parsey model:
http://petrovi.de/data/acl15.pdf
